fineract-provider/build.gradle

```diff
@@ -24,32 +24,33 @@ gradle clean tomcatrunwar
 project.ext.jerseyVersion = '1.17'
 
 buildscript {
-  ext {
-    openJPAVersion = '3.1.0'
-    jacocoVersion = '0.8.5'
-  }
-  repositories {
-  	 jcenter()
-  	 mavenCentral()
-  	 maven { url "https://plugins.gradle.org/m2/" }
-  }
-
-  dependencies {
-     classpath 'io.spring.gradle:dependency-management-plugin:1.0.9.RELEASE'
-     classpath 'com.bmuschko:gradle-tomcat-plugin:2.5',
-               'nl.javadude.gradle.plugins:license-gradle-plugin:0.11.0',
-               'org.zeroturnaround:gradle-jrebel-plugin:1.1.2',
-               'org.springframework.boot:spring-boot-gradle-plugin:2.2.4.RELEASE'
-     // below
-     classpath "org.apache.openjpa:openjpa:$openJPAVersion"
-     classpath 'com.radcortez.gradle:openjpa-gradle-plugin:3.1.0'
-     classpath 'gradle.plugin.org.nosphere.apache:creadur-rat-gradle:0.2.2'
-     classpath "com.github.spotbugs:spotbugs-gradle-plugin:2.0.1"
-      // Use Guava version 23+ as a workaround to spotbug intergration.
-      // See: https://github.com/spotbugs/spotbugs-gradle-plugin/issues/128#issuecomment-535864882
-     classpath 'com.google.guava:guava:28.1-jre'
-     classpath "gradle.plugin.com.github.andygoossens:gradle-modernizer-plugin:1.3.0"
-  }
+    ext {
+        openJPAVersion = '3.1.0'
+        jacocoVersion = '0.8.5'
+    }
+    repositories {
+        jcenter()
+        mavenCentral()
+        maven { url "https://plugins.gradle.org/m2/" }
+    }
+
+    dependencies {
+        classpath 'io.spring.gradle:dependency-management-plugin:1.0.9.RELEASE'
+        classpath 'com.bmuschko:gradle-tomcat-plugin:2.5'
+        classpath 'nl.javadude.gradle.plugins:license-gradle-plugin:0.11.0'
+        classpath 'org.zeroturnaround:gradle-jrebel-plugin:1.1.2'
+        classpath 'org.springframework.boot:spring-boot-gradle-plugin:2.2.4.RELEASE'
+        // below
+        classpath "org.apache.openjpa:openjpa:$openJPAVersion"
+        classpath 'com.radcortez.gradle:openjpa-gradle-plugin:3.1.0'
+        classpath "org.nosphere.apache:creadur-rat-gradle:0.6.0"
```

Comment:
This is interesting - why? Just newer version? And what does this have to do :smiling_imp: with FINERACT-883? Would you consider undoing this unrelated change here, and raising a separate (JIRA and?) PR for this?  Or does radle-git-properties somehow require this other RAT?